app/src/main/java/org/fossasia/pslab/communication/sensors/SSD1306.java

```diff
@@ -0,0 +1,470 @@
+package org.fossasia.pslab.communication.sensors;
+
+import android.content.Context;
+import android.hardware.usb.UsbManager;
+
+import org.fossasia.pslab.communication.ScienceLab;
+import org.fossasia.pslab.communication.peripherals.I2C;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.concurrent.TimeUnit;
+
+import static java.lang.Math.abs;
+
+/**
+ * Created by akarshan on 4/20/17.
+ */
+
+public class SSD1306 {
+    public int ADDRESS = 0x3C;
+    public HashMap<String, ArrayList> params = new HashMap<>();
+    public int NUMPLOTS = 0;
+    public String[] PLOTNAMES = {""};
+    public String name = "OLED Display";
+
+    private int _width = 128;
```

Comment:
take care of coding style (again). 
why did you used '_' here, don't create exact copy, adapt to coding style used in Java.
'_width' would be used if 'width' variable is reserved or there is a clash in variable name (If that be the case then its okay).